src/service/repo/anonymousMessageRepo.ts

```diff
@@ -119,13 +134,28 @@ async function setChannelAsRead(channelId: string): Promise<boolean> {
   }
 }
 
+async function deleteMessage(messageId: string) {
+  try {
+    const response = await anonymousApi.delete(baseUrl.deleteMessage(messageId));
+    if (response.status === 200) {
+      return Promise.resolve(response.data);
+    }
+
+    return Promise.reject(response.data?.status);
+  } catch (e) {
+    console.log(e);
+    return Promise.reject(e);
+  }
+}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

A new `deleteMessage` function has been added to handle deleting messages. This is a good addition, but there's a potential issue with error handling. The catch block simply logs the error and rejects the promise with the error object. This could leak sensitive information about the system. Instead, consider returning a custom error message.

```diff
-    console.log(e);
-    return Promise.reject(e);
+    console.error('Error deleting message:', e.message);
+    return Promise.reject('Error deleting message');
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->